Sources/XcodeGen/main.swift

```diff
@@ -21,7 +21,7 @@ func generate(spec: String, project: String, isQuiet: Bool, justVersion: Bool) {
         exit(1)
     }
 
-    let specPath = Path(spec).normalize()
+    let specPath = Path(URL(fileURLWithPath: spec).path).normalize()
```

Comment:
I didn't see any code in PathKit to do this.

Reply:
I am mistaken, I can change this to `absolute()`.

Reply:
Yep, absolute 👍